ddp-workspace/projects/ddp-dsm-ui/src/app/participant-list/models/data.model.ts

```diff
@@ -42,9 +42,15 @@ export class Data {
         });
       }
     }
+    const parsedFiles = [];
+    if(json.files){
+      for(const f of json.files){
+        parsedFiles.push(File.parse(f));
+      }
+    }
```

Comment:
**_#optional_** 
just fyi (don't have to fix)

```
 const parsedFiles = [];
  if(json.files){
    for(const f of json.files){
      parsedFiles.push(File.parse(f));
    }
  }
```
can be shortened like

` const parsedFiles = (json.files || []).map(file => File.parse(file)) `